.ci/run-extension-tests.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+# shellcheck disable=SC2016
+#
+# Copyright (c) 2012-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+set -e
+
+function installDeps() {
+    sudo pip install selenium
+    wget https://github.com/mozilla/geckodriver/releases/download/v0.29.0/geckodriver-v0.29.0-linux64.tar.gz
```

Comment:
Please create an variable for the geckodriver version, and use that variable in the URL. This makes it easier to read and update.